cs-rft/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.cloudslang.content</groupId>
     <artifactId>cs-rft</artifactId>
-    <version>0.0.4-SNAPSHOT</version>
+    <version>0.0.5-SNAPSHOT</version>
```

Comment:
this one should only be changed by the build system at release time, please change it back